src/analyses/ai_domain.cpp

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************\
+
+Module: Abstract Interpretation
+
+Author: Martin Brain, martin.brain@cs.ox.ac.uk
```

Comment:
IANL - but given that mostly these two new files contain code moved from other files, should the Author list at least still contain the Author from the original file? (Assuming that was accurate, which is likely debatable...)

Reply:
Fixed!

Reply:
This doesn't seem to be fixed?